doc/release/release-notes-19.2.md

```diff
@@ -4,29 +4,38 @@
 
 We are pleased to announce the release of TT Zephyr Platforms firmware version 19.2.0 🥳🎉.
 
-Major enhancements with this release include:
+Major enhancements with this release include upgrading to Zephyr 4.3.0, delivering configurable PCIe BAR support via the new `CntlInitV2()` flow, and introducing a standalone `tt_fwbundle` tooling suite.
 
 ## What's Changed
 
-<!-- Subsections can break down improvements by (area or board) -->
-<!-- UL PCIe -->
-<!-- UL DDR -->
-<!-- UL Ethernet -->
-<!-- UL Telemetry -->
-<!-- UL Debug / Developer Features -->
-<!-- UL Drivers -->
-<!-- UL Libraries -->
-
-<!-- Performance Improvements, if applicable -->
-<!-- New and Experimental Features, if applicable -->
-<!-- External Project Collaboration Efforts, if applicable -->
-<!-- Stability Improvements, if applicable -->
-<!-- Security vulnerabilities fixed? -->
-<!-- API Changes, if applicable -->
-<!-- Removed APIs, H3 Deprecated APIs, H3 New APIs, if applicable -->
-<!-- New Samples, if applicable -->
-<!-- Other Notable Changes, if applicable -->
-<!-- New Boards, if applicable -->
+### Platform & Dependencies
+- **Zephyr Upgrade**: Bumped the Zephyr manifest to the v4.3.0 release to pick up the latest upstream fixes and features.
+- **Devicetree Cleanups**: Relocated Blackhole SMC DTSI content into the vendor tree for clearer layering.
+
+### Drivers
+- **MSPI Upstreaming**: Migrated the MSPI controller and MSPI NOR flash drivers to their upstream Zephyr counterparts, reducing local maintenance.
+- **Clock Control Emulation**: Added an emulated clock control driver and bindings used by native-sim tests.
+- **DMA Reliability**: Switched the Blackhole DMA driver to RAM-to-Tensix transfers and enforced DT alignment so NOC DMA buffers meet 64-byte requirements.
+
+### Libraries & Firmware
+- **Power Management**: Refactored `bh_arc` power handling into `bh_power`, added L2 CPU enable/disable plumbing, and exposed the control through `tt_shell` with stricter error handling.
+- **PCIe Enhancements**: Adopted the new `CntlInitV2()` API, added structured logging, and aligned MSI message buffers for cleaner PCIe initialization.
+- **Firmware Tables**: Populated PCIe BAR size metadata across Blackhole board firmware tables to accompany the new BAR masks.
+
+### Tooling & Tests
+- **Firmware Bundle Utility**: Introduced the `tt_fwbundle` script with create, combine, list, diff, and extract commands, replacing the legacy `tt_boot_fs` bundle helpers.
```

Comment:
This isn't replacing anything legacy. This should be reworded.

It's just a separate tool for dealing with firmware bundles, while the other tool is for dealing with tt boot fs